src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/fake_quantize_decomposition_test.cpp

```diff
@@ -138,6 +138,7 @@ std::vector<std::pair<std::shared_ptr<ov::Node>, std::pair<std::string, std::str
     {std::make_shared<ov::op::v1::Convolution>(), {"Convolution", "Convolution,fakeQuantize"}},
 };
 
+#ifdef OPENVINO_ARCH_X86_64
```

Comment:
Minor: could you please move this line to L104 to compile whole namespace `legacyFuse` only on x64 platforms? Otherwise currently we create variables `operations, testValuesLegacyFuse` but use them only on x64 test instance